pldoc/static/js/pattern-library.js

```diff
@@ -3,6 +3,9 @@ require.config({
     paths: {
         jquery: "/public/js/jquery.min",
         svg4everybody: "/public/js/svg4everybody.min",
+        modernizr: "/public/js/modernizr-custom.js",
+        afontgarde: "/public/js/afontgarde.js",
+        edxicons: "/public/js/edx-icons.js"
     },
     shim: {
         'jquery': {
```

Comment:
Proposal from @andy-armstrong, add this to the shim after 'jquery':

    'afontgarde': {
        export: 'AFontGarde'
    }

You can then change the initialize.js to be:

    define([
    'afontgarde' // require this for this script to work
    ], function(AFontGarde) {
    AFontGarde('edx-icons', {
        // Check a few random icons to see if our font loads
        glyphs: '\uE621\uE622\uE623'
    });